aws/aws_s3_bucket/s3_static_website/main.tf

```diff
@@ -0,0 +1,61 @@
+# Summary: Setup a static website inside the S3 Bucket 
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 1.0.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.38"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+provider "aws" {
+  region = "us-east-1"
+  default_tags {
+    tags = {
+      cs_terraform_examples = "aws_s3_bucket/s3_static_website"
+    }
+  }
+}
+
+
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+# See also: [aws/aws_s3_bucket/simple] (https://github.com/ContainerSolutions/terraform-examples/tree/main/aws/aws_s3_bucket/simple)
+# Explanation: Sets a Bucket for Static Website Hosting
+# Now you have to add your files to the bucket index.html and also acces with this naming scheme , dependig on your region:
```

Comment:
Consider adding 2 [s3_bucket_object](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_object) referencing 2 files, `index.html` and `error.html`, which can also be added to this repo. A simple `index.html` file can be found [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html#step5-upload-index-doc)